cts/cli/regression.rules.exp

```diff
@@ -145,13 +148,16 @@ Rule cli-prefer-rule-dummy-date_spec-only-years satisfies conditions
 * Passed: crm_rule       - Verify date_spec rule with years is in effect
 =#=#=#= Begin test: Try to check a rule whose date_spec does not contain years= =#=#=#=
 Rule must either not use date_spec, or use date_spec with years= but not moon=
+Could not determine whether rule cli-prefer-rule-dummy-date_spec-without-years is expired
```

Comment:
Looking at these two next to each other, it'd be nice if the top message also said "Rule cli-prefer-rule-dummy-date_spec-without-years must either...".

Reply:
Good point. The other three "failed to check rule" messages all include the rule ID.